server/service/src/sync/translations/item.rs

```diff
@@ -69,7 +69,9 @@ fn to_legacy_ven_category(ven_category: VENCategory) -> String {
 }
 
 pub(crate) fn ordered_simple_json(text: &str) -> Result<String, serde_json::Error> {
-    let json: serde_json::Value = serde_json::from_str(text)?;
+    let mut json: serde_json::Value = serde_json::from_str(text)?;
+    // Saving dose_picture 'picture' type as incoming 'empty string' causes issues in integration tests
+    json.as_object_mut().unwrap().remove("dose_picture");
```

Comment:
Is unwrap() safe here? Should we consider what happens if it's not a json or `null` which I have seen sync'd once, perhaps not for item but still... will it block sync permanently if this happened??

Reply:
technically not possible but i 100% agree we shouldn't be unwrapping, very nice catch thanks @jmbrunskill